### ISSUE TITLE
Cannot observe Route#controller

### ISSUE THREAD

**Issue by teddyzeenny** [emberjs/ember-inspector#100]
Mandatory setters prevent us form observing `Route#controller` and therefore we cannot show it in the Object Inspector.

The main reason is [the controller is set without `Ember.set`](https://github.com/emberjs/ember.js/blob/master/packages/ember-routing/lib/system/route.js#L468) in `Route#setup`

This has 3 possible solutions:
- Add the ability to temporarily disable mandatory setters so we can observe whatever we want in the Object Inspector
- Use `Ember.set` [here](https://github.com/emberjs/ember.js/blob/master/packages/ember-routing/lib/system/route.js#L468).
- Remove `Route#controller` from the Object Inspector by marking it as private (which it's not so I'm not a fan of doing this).

**Comment by locks** [comment]
Can we get some input @rwjblue @krisselden ?

**Comment by rwwagner90** [comment]
@teddyzeenny @locks @rwjblue @krisselden this has been waiting on feedback for several years. Any updates here or should we close as wontfix?

**Comment by rwwagner90** [comment]
Since there has been no feedback, I am closing for now. Please feel free to reopen, if there is still a need for this.
